back/engines/commercial/idea_custom_fields/app/controllers/idea_custom_fields/web_api/v1/admin/idea_custom_fields_controller.rb

```diff
@@ -94,7 +94,7 @@ def update_fields!(page_temp_ids_to_ids_mapping, option_temp_ids_to_ids_mapping,
             option_temp_ids_to_ids_mapping_in_field_logic = update_options! field, options_params, errors, index
             option_temp_ids_to_ids_mapping.merge! option_temp_ids_to_ids_mapping_in_field_logic
           end
-          field.move_to_bottom
```

Comment:
This appeared to be the root cause of the bad reordering as this function was obviously reordering within the context of all fields not just CustomForm fields

Reply:
I'm very surprised that `move_to_bottom` does not work with a scope. This sounds like quite a bug for this gem. Is `set_list_position` a better solution than `insert_at `?

Reply:
Yes in this case it is, as it set the position without reordering anything else.